.github/workflows/daily-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: daily-tests
+
+permissions:
+  contents: read
+
+on:
+  schedule:
+    # We run these tests on a daily basis (at a time slightly offset from the
+    # top of the hour), as their runtime is either too long for the usual per-PR
+    # CI, or because they rely on external 3rd party services that can be flaky.
+    - cron: '15 18 * * *'
+
+jobs:
+  ignored:
+    # Run slower tests we make opt-in with the #[ignore] annotation.
+    name: Ignored tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # test a bunch of toolchains on ubuntu
+        rust:
+          - stable
+          - beta
+          - nightly
+        os: [ubuntu-20.04]
+        # but only stable on macos/windows (slower platforms)
+        include:
+          - os: macos-latest
+            rust: stable
+          - os: windows-latest
+            rust: stable
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install ${{ matrix.rust }} toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.rust }}
+
+      - name: Run ignored tests
+        run: cargo test -- --ignored
```

Comment:
Personally I would keep CI running all these tests every time, as 5-10 seconds feels very minor there. Especially if these tests are rarely going to be run locally during development.

Reply:
@djc Want to argue your case? :laughing: I put this branch together based on your feedback in Discord. Personally I'm used to runtimes in O(minutes) but maybe accepting that as normal is stockholm syndrome from time spent w/ slower languages?

Reply:
fwiw, I think if we drop the slow `test_too_many_path_calls` test, and keep `test_too_many_path_calls_mini` running all the time, we're just trying to decide whether the bettertls suites should run all the time, or opt-in.

Reply:
FWIW I'm completely fine with running these in default CI, my complaint was about local iteration. Agreed that a few seconds more or less doesn't matter for CI.

Reply:
:+1: Cool, I'll move these from the daily tests to default CI. We can keep daily tests for the cargo hack bits.

Reply:
Fair point, looks to be [~12s](https://github.com/cpu/webpki/actions/runs/6166808649/job/16736859734). I'll drop the daily tests workflow and fold that into default CI too.

Reply:
Just tried this -- 14s currently. #158 will take it to ~30s.

I also noticed that `cargo test --no-default-features --features alloc` doesn't build currently. Quite a lot of tests have a (quite reasonable, I think) dependency on std.

Reply:
> I also noticed that `cargo test --no-default-features --features alloc` doesn't build currently. Quite a lot of tests have a (quite reasonable, I think) dependency on std.

Will be good to get that fixed.